File: module_home_tab_task/src/main/java/com/hengwei/module_home_tab_task/ui/main/adapter/ExpandAdapter.java
package com.hengwei.module_home_tab_task.ui.main.adapter;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.hengwei.module_home_tab_task.R;
import com.hengwei.module_home_tab_task.bean.ExpandItem;

import java.util.List;

/***
 * @author 86153
 * 扩展的子列表
 */
public class ExpandAdapter extends BaseQuickAdapter<ExpandItem, BaseViewHolder> {

    public ExpandAdapter(List<ExpandItem> data) {
        super(R.layout.tab_task_item_main, data);
    }

    @Override
    protected void convert(BaseViewHolder baseViewHolder, ExpandItem expandItem) {
        baseViewHolder.setText(R.id.textTitle,expandItem.getExpandString());
    }
}

File: module_home_tab_task/src/main/java/com/hengwei/module_home_tab_task/ui/main/adapter/ExpandAdapter.kt
package com.hengwei.module_home_tab_task.ui.main.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.hengwei.module_home_tab_task.R
import com.hengwei.module_home_tab_task.bean.ExpandItem

/***
 * @author 86153
 * 扩展的子列表
 */
class ExpandAdapter(data: MutableList<ExpandItem>) : BaseQuickAdapter<ExpandItem, BaseViewHolder>(R.layout.tab_task_item_expand, data) {
    override fun convert(baseViewHolder: BaseViewHolder, expandItem: ExpandItem) {
        baseViewHolder.setText(R.id.textTitle, expandItem.expandString)
    }
}